plugins/auth-backend/src/identity/DatabaseKeyStore.ts

```diff
@@ -42,6 +42,7 @@ export class DatabaseKeyStore implements KeyStore {
 
     await database.migrate.latest({
       directory: migrationsDir,
+      tableName: 'knex_migrations_auth',
```

Comment:
Would ye be alright with removing this change from the PR? Since it's not backwards-compatible and we likely want to solve it in a different way.

Reply:
Sure thing. @dariddler is on vacation and I don't have access to his repo to clean it up (we will have a more production like git setup in the future 😬), would you mind stripping it, I think you as code owners should be able to push to the PR. Otherwise I can re-create some time later.

Reply:
We can't : /, unless this is enabled: https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests

Reply:
absolutely!

I think we'll be looking at a different implementation-path of these plugin-backends once vacation is over either way :)

Reply:
never mind, @dariddler is fixing it